recipes/userver/all/conanfile.py

```diff
@@ -0,0 +1,655 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import get, copy, export_conandata_patches, apply_conandata_patches
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.53.0"
+
+class UserverConan(ConanFile):
+    name = 'userver'
+    description = 'The C++ Asynchronous Framework'
+    topics = ('framework', 'coroutines', 'asynchronous')
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'https://userver.tech/'
+    license = 'Apache-2.0'
+    package_type = "library"
+
+    settings = 'os', 'arch', 'compiler', 'build_type'
+    options = {
+        'shared': [True, False],
+        'fPIC': [True, False],
+        'lto': [True, False],
+        'with_jemalloc': [True, False],
+        'with_mongodb': [True, False],
+        'with_postgresql': [True, False],
+        'with_postgresql_extra': [True, False],
+        'with_redis': [True, False],
+        'with_grpc': [True, False],
+        'with_clickhouse': [True, False],
+        'with_rabbitmq': [True, False],
+        'with_utest': [True, False],
+        'namespace': ['ANY'],
+        'namespace_begin': ['ANY'],
+        'namespace_end': ['ANY'],
+    }
+
+    default_options = {
+        'shared': False,
+        'fPIC': True,
+        'lto': True,
+        'with_jemalloc': False,
+        'with_mongodb': True,
+        'with_postgresql': True,
+        'with_postgresql_extra': False,
+        'with_redis': True,
+        'with_grpc': True,
+        'with_clickhouse': True,
+        'with_rabbitmq': True,
+        'with_utest': True,
+        'namespace': 'userver',
+        'namespace_begin': 'namespace userver {',
+        'namespace_end': '}',
+        'mongo-c-driver/*:with_sasl': 'cyrus',
+    }
+
+    @property
+    def _source_subfolder(self):
+        return 'source'
+
+    @property
+    def _build_subfolder(self):
+        return os.path.join(self.build_folder, 'userver')
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires('boost/1.79.0', transitive_headers=True)
```

Comment:
Is it stuck on old versions for upstreams reasons?

Reply:
I create package locally with such versions and run all internal tests. If I need bump core dependency I need to rebuild package with all tests and run it. I don't have enough time sometimes for this :) There is only one reason :)